### ISSUE TITLE
Black and white

### ISSUE THREAD

**Issue by zmartin0028** [AlessandroYorba/Alduin#23]
Hi, excellent colorscheme. I've been trying to get it installed and I'm having some trouble. I have installed many other colorschemes before successfully. 

I placed the alduin.vim file in my .vim/colors directory, but when I select it as the default theme, I get white text on my dark background. I've used other 256 colorschemes before, and I tested my terminal color output just to be sure. 

Does alduin.vim have a dependency on another file, by chance? Is this something that you've dealt with before?

**Comment by AlessandroYorba** [comment]
@zmartin0028 sorry to hear about the issue. There are no other dependencies other than having a terminal that supports 256 colors and it sounds like you've got that covered. It almost sounds like syntax is disabled. 
Can you tell me more about your environment?

**Comment by zmartin0028** [comment]
First of all, thanks for your willingness to help. I've only been using Vim as my editor of choice for a couple of months, so I'm still learning new things on a daily basis. And in the grand scheme, I'm still fairly new to the Linux world. 

I did a bit more digging and it seems that, while my terminal does support 256 colors, it wasn't configured to do so by default. This resulted in a test that produced 256 colors, but an environment that didn't support it. I've made some environmental tweaks and I'm going to reinstall your colorscheme. I'll update in a bit, but I'm confident that it will work now. 

Also worth noting that, while I have used 256 color themes before, they almost certainly didn't look the way that they should have! I'll have to try them again and see how they were intended to look.
